src/popup/utils/helper.js

```diff
@@ -61,7 +61,7 @@ export const checkAddress = (value) => Crypto.isAddressValid(value, 'ak')
 
 export const checkAddressOrChannel = (value) => checkAddress(value) || Crypto.isAddressValid(value, 'ch');
 
-export const checkAensName = (value) => value.endsWith('.chain');
+export const checkAensName = (value) => /^[a-z]+\.chain$/i.test(value);
```

Comment:
We have to refactor this: names can include also numbers and probably all UTF-8 characters.